doc/search-metrics.org

```diff
@@ -96,13 +97,19 @@ GET /ctia/judgement/topn?aggregate-on=observable.value&limit=20&from=2020-01-01&
 #+END_SRC
 
 #+BEGIN_SRC javascript
-[
- {"key": "abuse.ch", "value": 62},
- {"key": "8.8.8.8", "value": 62},
- {"key": "4964ab7e8d5959bb42b8ef78582082686dbe6565b010824785bc44595d8ebeee", "value": 38},
- {"key": "https://www.badbabbad.net/worse.html", "value": 25},
- ...
-]
+{"data": {"observable":
+           {"value":
```

Comment:
I'm not sure why the level `value` is needed here.
aren't all `topn` results a list?
So I think you could expect that observale value would be directly the list.
to prevent a bit of redundancy.

Reply:
:-)
`value` in this example is the nested field of the object [`observable`](https://github.com/threatgrid/ctim/blob/master/src/ctim/schemas/common.cljc#L321). In this example we queried a topn on `observable.value`. On `observable.type` the result will look like:
``` javascript 
{
 "data": {"observable": "type": [...]}
}
```